src/renderer.ts

```diff
@@ -43,6 +43,39 @@ export class Renderer {
 		});
 	}
 
+	public async renderSignature(document: vscode.TextDocument, signatureHelp: vscode.SignatureHelp): Promise<string> {
+
+		if (signatureHelp.signatures.length === 0) {
+			return '';
+		}
+
+		let parts: any[] = [];
+		const activeSignature: any[] = [];
+
+		signatureHelp.signatures.forEach((signatureInformation: vscode.SignatureInformation, index: number) => {
+			let pushTo;
+			if (signatureHelp.activeSignature === index) {
+				pushTo = activeSignature;
+			} else {
+				pushTo = parts;
+			}
+			pushTo.push(this.getMarkdown(
+				`${String(index + 1).padStart(2, '0')}/${String(signatureHelp.signatures.length).padStart(2, '0')} ${signatureHelp.activeSignature === index ? '🟩' : '⬛'} \`\`\`${signatureInformation.label}\`\`\``));
+			if (signatureInformation.documentation) {
+				pushTo.push(this.getMarkdown(signatureInformation.documentation as vscode.MarkdownString));
+			}
+		});
+		parts = ['', ...activeSignature, ...parts];
+
+		const markdown = parts.join('\n___\n');
```

Comment:
For styling, there should be a more prominent separation between each signature than between parts of an individual signature. You may need to use css for this

Reply:
Any hint on how to use CSS?